<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>875</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="cw">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDockWidget" name="dw">
      <property name="maximumSize">
       <size>
        <width>524287</width>
        <height>524287</height>
       </size>
      </property>
      <property name="windowTitle">
       <string>fcVM</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>396</width>
             <height>757</height>
            </rect>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>371</width>
                <height>721</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QGroupBox" name="materialBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>210</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Material parameters (additional to FEM_Material object)</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Yield Stress</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <widget class="QLineEdit" name="GXinput">
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>27</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>0.0</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="GYinput">
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>27</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>0.0</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="GZinput">
                      <property name="maximumSize">
                       <size>
                        <width>150</width>
                        <height>27</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>-10.0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>(m/s2)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>(MPa)</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Gravity_x</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Gravity_y</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Gravity_z</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="YSinput">
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>240.0</string>
                    </property>
                    <property name="frame">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QGroupBox" name="actionBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Actions</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="3">
                   <widget class="QPushButton" name="quitBtn">
                    <property name="text">
                     <string>Quit</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QPushButton" name="startBtn">
                    <property name="text">
                     <string>Start</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="resetBtn">
                    <property name="text">
                     <string>Reset</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="saveBtn">
                    <property name="text">
                     <string>Save</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QGroupBox" name="outputBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Output options </string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="incrRbtn">
                    <property name="text">
                     <string>Incremental displacements (failure mechanism)</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="totalRbtn">
                    <property name="text">
                     <string>Total displacements</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QGroupBox" name="controlBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>500</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Control parameters (press Reset for defaults)</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="scale_1">
                    <property name="text">
                     <string>2.0</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Error</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="error">
                    <property name="text">
                     <string>1.0e-3</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QLineEdit" name="max_iter">
                    <property name="text">
                     <string>20</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="steps">
                    <property name="text">
                     <string>10</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Max. Iter.</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Relax.</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Scale1</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Scale2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3">
                   <widget class="QLineEdit" name="scale_2">
                    <property name="text">
                     <string>1.2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Steps</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="3">
                   <widget class="QLineEdit" name="relax">
                    <property name="text">
                     <string>1.2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Scale3</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="scale_3">
                    <property name="text">
                     <string>1.2</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QFrame" name="filler">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
